src/m365/spo/commands/list/list-roleinheritance-break.spec.ts

```diff
@@ -0,0 +1,243 @@
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('break role inheritance on list by title', (done) => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if ((opts.url as string).indexOf('/_api/web/lists/getbytitle(\'test\)/breakroleinheritance(false)') > -1) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, {
+      options: {
+        debug: true,
+        webUrl: 'https://contoso.sharepoint.com',
+        listTitle: 'test'
+      }
+    }, () => {
+      try {
+        assert(true);
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('break role inheritance on list by title and clear all permissions', (done) => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if ((opts.url as string).indexOf('/_api/web/lists/getbytitle(\'test\)/breakroleinheritance(true)') > -1) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, {
+      options: {
+        debug: true,
+        webUrl: 'https://contoso.sharepoint.com',
+        listTitle: 'test',
+        clearExistingPermissions: true
+      }
+    }, () => {
+      try {
+        assert(true);
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('break role inheritance on list by id', (done) => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if ((opts.url as string).indexOf('/_api/web/lists(guid\'202b8199-b9de-43fd-9737-7f213f51c991\')/breakroleinheritance(false)') > -1) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, {
+      options: {
+        debug: true,
+        webUrl: 'https://contoso.sharepoint.com',
+        listId: '202b8199-b9de-43fd-9737-7f213f51c991'
+      }
+    }, () => {
+      try {
+        assert(true);
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('break role inheritance on list by id and clear all permissions', (done) => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if ((opts.url as string).indexOf('/_api/web/lists(guid\'202b8199-b9de-43fd-9737-7f213f51c991\')/breakroleinheritance(true)') > -1) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, {
+      options: {
+        debug: true,
+        webUrl: 'https://contoso.sharepoint.com',
+        listId: '202b8199-b9de-43fd-9737-7f213f51c991',
+        clearExistingPermissions: true
+      }
+    }, () => {
+      try {
+        assert(true);
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('list role inheritance break command handles reject request correctly', (done) => {
+    const err = 'Invalid request';
```

Comment:
Let's use a different message here so that we can differentiate between a valid request that we're failing on purpose and an invalid request